src/queries.jl

```diff
@@ -301,8 +299,8 @@ for (tagsymbol, tagvariant) in pairs(tag_types)
     args = (:a, :b, :c, :d, :e, :f, :g)[1:length(sig)]
     argssig = [:($a::$t) for (a,t) in zip(args, sig)]
 
-    eval(quote function tag!(ref::RegRef, $(argssig...))
-        tag!(ref, ($tagvariant)($(args...)))
+    eval(quote function tag!(ref::RegRef, $(argssig...); tag_time::Union{Float64, Nothing}=nothing, id::Union{Int128, Nothing}=nothing)
```

Comment:
these can probably just be `;kwargs...` so that we do not need to edit this line each time we add new keyword arguments